variables/lifetimeSupplyCalculator.js

```diff
@@ -1 +1,12 @@
-// Enter code below
\ No newline at end of file
+// Enter code below
+var me={ 
+	currentAge:30,
+	maxAge:99,
+	estimatedAmount:10
+}
+function lifetimeSupply(){
+	var lifeSpan=(me.maxAge-me.currentAge); // number of years till death 
+	var numDays=(lifeSpan*365); //calculate the number of days till i die 
```

Comment:
If I was going to be really picky you should try and use spaces around `=` and math operators like `+`, `-`, `*` & `/`
So 
```
var numDays=(lifeSpan*365);
```
becomes
```
var numDays = ( lifeSpan * 365 );
```
It just makes things a little nice to read 😃